content/docs/setup-and-administration/configuring-app-registrations-for-non-interactive-use/index.md

```diff
@@ -0,0 +1,168 @@
+
+{{< warning >}}
+**Warning:** You may only create and manage `App Registrations` with a **Root User**.
+{{< /warning >}}
+
+When enabling non-interactive access to RKVST, you ***must*** create your first App Registration in the **RKVST UI.**
+
+### Using the RKVST UI (Required for First-Time Setup)
+
+1. As a Root User visit the APP REGISTRATIONS tab in the `Manage RKVST` Interface
+2. Click CREATE APP REGISTRATION.
+3. Enter any display name you like.
+
+ You may also optionally add any Custom Claims at this step, you must ensure they do not start with `jit_` or use of the [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+
+4. Click CREATE APP REGISTRATION. The response will include the CLIENTID and SECRET required by the archivist token endpoint.
+
+{{< caution >}}
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+{{< /caution >}}
+
+5. Now you have created your App Registration, follow the steps further below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
+
+{{< note >}}
+**Note:** By default, newly created Applications will always have a Non-Root User permission to the API. To give an Application Root User priviliges so it can be used to administrate RKVST, including adding and managing new App Registrations, please refer to our [Tenancies API](../../api-reference/tenancies-api).
+{{< /note >}}
+
+### Using the App Registrations API
+
+The following assumes you have at least one `App Registration` that has already been configured with Root User permissions and that you are comfortable generating tokens and using the RKVST API.
+
+If you do not yet have an App Registration configured please follow [the first-time setup guide](./#using-the-rkvst-ui-(required-for-first-time-setup)) to get started.
+
+1. Define your new Application JSON and save it to a file locally. e.g.
+
+```json
+{
+    "display_name": "TrafficLight101",
+    "custom_claims": {
+      "serial_number": "TL1000000101",
+      "has_cyclist_light": "true"
+    }
+}
+```
+
+2. Generate a token using your pre-existing `App Registration` details
+
+```bash
+RESPONSE=$(curl \
+    https://app.rkvst.io/archivist/iam/v1/appidp/token \
+    --data-urlencode "grant_type=client_credentials" \
+    --data-urlencode "client_id=${CLIENTID}" \
+    --data-urlencode "client_secret=${SECRET}")
+
+TOKEN=$(echo -n $RESPONSE | jq .access_token | tr -d '"')
+```
+
+and save it locally to a bearer_token file.
+
+3. Submit your new Application JSON to the App Registration API Endpoint 
+
+```bash
+curl -X POST \
+     -H "@$BEARER_TOKEN_FILE" \
+     -H "Content-Type: application/json" \
+     -d "@/path/to/jsonfile" \
+     $URL/archivist/iam/v1/applications
+```
+
+You should see a response with details about the App Registration's `CLIENT ID` and `SECRET`:
```

Comment:
(minor/moderate) Can we please use CLIENTID or CLIENT_ID in the quoted sections ? Those are meant to be conveniences and queues for shell variables. If it has a space in it then anyone building up code using the fragments is going to need to remove the space. Assuming these docs are for developers making automation and scripts, 'spaces in variables' is a bit of a face slap imo

Reply:
Done sticking with CLIENT_ID - this should be updated across the doc